avro-codegen/src/main/java/com/linkedin/avro/codegen/ToolArgs.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avro.codegen;
+
+import com.beust.jcommander.Parameter;
+import com.linkedin.avro.compatibility.AvroVersion;
+import java.io.File;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+
+public class ToolArgs {
+  @Parameter(names = "-src", required = true, description = "files or folders to generate code for", converter = StringToFileConverter.class)
+  private Set<File> src;
+  @Parameter(names = "-include", description = "include patterns. default is \"**/*.avsc\"")
+  private Set<String> includes = new HashSet<>(Collections.singletonList("**/*.avsc"));
+  @Parameter(names = "-importable", description = "patterns for importable schema files. default is \"**/*.*\"")
+  private Set<String> importablePatterns = new HashSet<>(Collections.singletonList("**/*.*"));
+  //if true will go fishing on the classpath for generated specific record classes of missing schemas
```

Comment:
Can these comments be javadoc comments so I can see them popup in my lazy person's ide.